src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@GeertvanHorrik There is already a Cake Alias for GitLink, so we will need to understand how these changes are going to affect that alias, so that we know what changes need to be made.